README.md

```diff
@@ -1,22 +1,24 @@
-# @stripe/react-stripe
-
-[![build status](https://img.shields.io/travis/stripe/react-stripe/master.svg?style=flat-square)](https://travis-ci.org/stripe/react-stripe)
-[![npm version](https://img.shields.io/npm/v/@stripe/react-stripe.svg?style=flat-square)](https://www.npmjs.com/package/@stripe/react-stripe)
-
-> React components for Stripe.js and Stripe Elements
+## React Stripe
 
 React Stripe is a thin React wrapper around
 [Stripe Elements](https://stripe.com/docs/elements). It allows you to add
 Elements to any React app.
 
-## Docs
+[![build status](https://img.shields.io/travis/stripe/react-stripe/master.svg?style=flat-square)](https://travis-ci.org/stripe/react-stripe)
+[![npm version](https://img.shields.io/npm/v/@stripe/react-stripe.svg?style=flat-square)](https://www.npmjs.com/package/@stripe/react-stripe)
+
+# Docs
```

Comment:
I feel like this could use a "Getting Started" section, even if it just links to the React Stripe docs.

Reply:
Maybe this section could be called "Getting Started" instead of "Docs"?